src/main/java/dk/cngroup/lentils/service/FinalPlaceService.java

```diff
@@ -5,12 +5,18 @@
 import dk.cngroup.lentils.repository.FinalPlaceRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
 
+import java.time.LocalDateTime;
 import java.util.List;
 
 @Service
 public class FinalPlaceService {
 
+    private static final String FINAL_PLACE_ENTITY = "finalPlace";
+    private static final String OPENING_TIME_PROPERTY = "openingTime";
```

Comment:
Jen drobnost, možná bych to pojmenoval jako OPENING_TIME_FIELD, pokud PROPERTY už není nikde jinde, ať to má stejné pojmenování co opužívá Spring.

Reply:
pri validaci unikatniho user name puzivame taky PROPERTY -> muzu to zmenit i tam na FIELD

Reply:
Tak to jo. Nevšiml jsem si. Tak už nech PROPERTY, ať se to nemíchá dohromady v tomhle PR.